--- spack-src/contrib/metis/GKlib/string.c	2025-10-26 13:59:26.000000000 +0000
+++ spack-src/contrib/metis/GKlib/string.c	2025-10-26 15:51:02.000000000 +0000
@@ -12,8 +12,9 @@
 \version $Id: string.c 10711 2011-08-31 22:23:04Z karypis $
 */
 /************************************************************************/
-
+#define _XOPEN_SOURCE
 #include "GKlib.h"
+#include <time.h>
 
 
 /************************************************************************/
